openedx/core/djangoapps/util/management/commands/run_aggregator.py

```diff
@@ -0,0 +1,272 @@
+# -*- coding: utf-8 -*-
+"""
+Performance tests for completion aggregator.
+"""
+
+import datetime
+import math
+import random
+from timeit import default_timer
+
+import pytz
+from completion.models import BlockCompletion
+from django.core.management.base import BaseCommand, CommandError
+from django.db import connection, reset_queries
+import numpy
+from student.tests.factories import UserFactory, CourseEnrollmentFactory
+from xmodule.modulestore import ModuleStoreEnum
+from xmodule.modulestore.django import SignalHandler, modulestore
+
+
+class Command(BaseCommand):
+    """ Run performance tests for completion aggregator. """
+
+    help = "Run performance tests for completion aggregator."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'test',
+            help='Test to run.'
+        )
+        parser.add_argument(
+            '--course-breadth',
+            nargs=4,
+            type=int,
+            default=(3, 3, 3, 3),
+            help='Number of blocks at the first four levels in course.'
+        )
+        parser.add_argument(
+            '--learners',
+            type=int,
+            default=2,
+            help='Number of learners.'
+        )
+        parser.add_argument(
+            '--completions',
+            type=int,
+            default=30,
+            help='Number of completions to perform.'
+        )
```

Comment:
Nit: Include default values in help text.  Unfortunately, running --help doesn't print that information otherwise.

Reply:
Done.